erpnext/projects/doctype/project/project.py

```diff
@@ -212,7 +212,7 @@ def send_welcome_email(self):
 
 	def on_update(self):
 		self.notify()
-
+		update_estimated_time(self.name)
```

Comment:
move this to after_save hook so that the save doesn't slow down because of this

Reply:
used after_save method to update estimated_time in project doctype